webapp/channels/src/components/admin_console/custom_terms_of_service_settings/custom_terms_of_service_settings.tsx

```diff
@@ -200,8 +202,18 @@ export default class CustomTermsOfServiceSettings extends AdminSettings<Props, S
                 <BooleanSetting
                     key={'customTermsOfServiceEnabled'}
                     id={'SupportSettings.CustomTermsOfServiceEnabled'}
-                    label={<FormattedMessage {...messages.enableTermsOfServiceTitle}/>}
-                    helpText={<FormattedMarkdownMessage {...messages.enableTermsOfServiceHelp}/>}
```

Comment:
only this has the internal link, so this must be changed others can be kept as is. See this PR for reference https://github.com/mattermost/mattermost/pull/28635

Reply:
@M-ZubairAhmed Currently I don't see any internal link here or perhaps taking reference from the #28635 do you want me to include this in the values 
```
a: (chunks: string) => (
                                <Link to='/admin_console/environment/file_storage'>
                                    {chunks}
                                </Link>
                            ) 
```
for every FormattedMessage tag I used ` <b></b>` tags in the values are you fine with this or should i remove them as well?

Reply:
you would have to check if each message if it contains markdown or not. If it does then you should apply suitable HTML replacement like for bold -> `<b>`

Reply:
Okay ..I will check that.
What about the link that you were mentioning ,I don't see any internal link that you were talking about?

Reply:
i meant that in `enableTermsOfServiceHelp` i dont any bold markdown but you have added that! i can see only link being used there. thats why i asked you to recheck all https://github.com/mattermost/mattermost/pull/28690#discussion_r1798669476